lisa/base_tools/uname.py

```diff
@@ -8,10 +8,34 @@
 from lisa.util import LisaException
 
 
+@dataclass
+class KernelVersion:
```

Comment:
Did you check _parse_version() and VersionInfo in operating_system.py? I think it can give you what you are looking for here.

Reply:
I think I might be able to use VersionInfo class. I will give it a try.